### ISSUE TITLE
Use Finder Default Cmd+Shift+. To Toggle Hidden Files

### ISSUE THREAD

**Issue by williamconnolly** [marta-file-manager/marta-issues#538]
Finder allows you to toggle hidden files by pressiing <kbd>Cmd+Shift+.</kbd>. It doesn't look like this keybinding format is supported by Marta.

My guess as to why this isn't working:

The <kbd>.</kbd>, dot/period/full stop character isn't supported (or has a different format that I'm unaware of in Marco's processor. I've tried "Dot" and "Period" as well. 

Any advice? Love Marta so far, keep up the great work!

**Comment by yanex** [comment]
It seems like this is already fixed in Marta 0.6.2. Please check if the following configuration works for you:

```
keyBindings {
    "Cmd+Shift+." {
        id "core.toggle.show.hidden.files"
    }
}
```

**Comment by yanex** [comment]
`Cmd+Shift+.` became the default hotkey for toggling hidden file visibility, so I'll close the issue. Thanks for reporting!
